pkg/resolver/searchSubscription.go

```diff
@@ -0,0 +1,46 @@
+// Copyright Contributors to the Open Cluster Management project
+package resolver
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/stolostron/search-v2-api/graph/model"
+)
+
+func SearchSubscription(ctx context.Context, input []*model.SearchInput) (<-chan []*SearchResult, error) {
+	ch := make(chan []*SearchResult)
+
+	// You can (and probably should) handle your channels in a central place outside of `schema.resolvers.go`.
+	// For this example we'll simply use a Goroutine with a simple loop.
+	go func() {
+		// Handle deregistration of the channel here. Note the `defer`
+		defer close(ch)
+
+		for {
+			// Send the search results every 10 seconds.
+			time.Sleep(10 * time.Second)
+			fmt.Println("SearchSubscription new poll interval")
+			
+			// TODO 2nd return item is error... needs to be handled
+			searchResult, _ := Search(ctx, input)
```

Comment:
Are you saying we're getting an error here or just that we need to implement handling? Because I'm not encountering any errors in testing

Reply:
Just that I need to handle/log it.